sdk/src/main/java/com/mapbox/maps/renderer/WorkerHandlerThread.kt

```diff
@@ -12,6 +12,9 @@ internal class WorkerHandlerThread {
   @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
   internal var handler: Handler? = null
 
+  internal var started = false
```

Comment:
Instead of recording the state, we can add a `get()` and return `handlerThread.isAlive`

Reply:
nice idea but a really small one
if you don't mind, will leave as is for now, I have a bit of plans to revisit rendering logic a bit in future, I'll do that change in another PR

Reply:
as was pushing some changes also applied your suggestion.